legendary/cli.py

```diff
@@ -779,6 +779,8 @@ def _launch_origin(self, args):
                          f'wrapper in the configuration file or command line. See the README for details.')
             return
 
+        # You cannot launch a URI without start.exe, this applies to windows as well
+        command.append('start.exe ')
```

Comment:
Do not have a trailing space or `.exe` here. Technically the syntax here is also wrong for Windows (on Windows an empty "title" argument would be required before the URL), so remove that comment.